app/entry.server.tsx

```diff
@@ -9,18 +9,9 @@
 import { PassThrough } from 'node:stream'
 import { Response, type AppLoadContext, type EntryContext } from '@remix-run/node'
 import { RemixServer } from '@remix-run/react'
-import * as Sentry from '@sentry/remix'
 import isbot from 'isbot'
 import { renderToPipeableStream } from 'react-dom/server'
 
-if (typeof process.env.SENTRY_DSN === 'string') {
-  Sentry.init({
-    dsn: process.env.SENTRY_DSN,
-    tracesSampleRate: 0.1,
-    enabled: process.env.NODE_ENV === 'production',
-  })
-}
-
```

Comment:
Actually... I bet we can delete this file too. It's only in the other apps that we're doing the cache header thing that keeps this file around.